lib/capistrano/dkdeploy/typo3_cms.rb

```diff
@@ -21,13 +21,13 @@
     set :create_caretaker_instance_keys_path, -> { File.join(shared_path, 'config', 'create_caretaker_instance_keys.php') }
     set :caretaker_instance_keys_path, -> { File.join(shared_path, 'config', 'caretaker_instance_keys.php') }
 
-    # Use custom Composer autoload
     set :typo3_environment_cli, {
-      TYPO3_COMPOSER_AUTOLOAD: 1,
+      CUSTOM_VARIABLE: 1,
```

Comment:
Please remove `CUSTOM_VARIABLE`.
If needed for tests, please move it to test application.